pkg/controller/migplan/migplan_controller.go

```diff
@@ -71,6 +73,16 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Watch for changes to deployment registry referenced by MigPlan
+	err = c.Watch(
+		&registryHealth{
+			hostClient: mgr.GetClient(),
+			Interval: time.Second*5},
```

Comment:
Am I correct in thinking that this watch will only list MigPlans with a MigMigration `in progress` by using a label selector?

Reply:
I found out later in review that we aren't using a labelselector for this list operation. Need to understand why.